content/en/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Kubernetes API Server Bypass Risks
+description: >
+  Security architecture information relating to the API server and other components
+content_type: concept
+---
+
+<!-- overview -->
+
+The Kubernetes API server is the main point of entry to a cluster for external parties 
+(users and services) interacting with it. 
+
+As part of this role, the API server has several key built-in security controls, such as
+audit logging and {{< glossary_tooltip text="admission controllers" term_id="admission-controller" >}}. However, there are ways to modify the configuration
+or content of the cluster that bypass these controls.
+
+This page describes the ways in which the security controls built into the
+Kubernetes API server can be bypassed, so that cluster operators
+and security architects can ensure that these bypasses are appropriately restricted.
+
+<!-- body -->
+
+## Static Pods {#static-pods}
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on each node loads and
+directly manages any manifests that are stored in a named directory or fetched from
+a specific URL as [*static Pods*](/docs/tasks/configure-pod-container/static-pod) in
+your cluster. The API server doesn't manage these static Pods. An attacker with write
+access to this location could modify the configuration of static pods loaded from that
+source, or could introduce new static Pods.
+
+Static Pods are restricted from accessing other objects in the Kubernetes API . For example,
+you can't configure a static Pod to mount a Secret from the cluster. However, these Pods can
+take other security sensitive actions, such as using `hostPath` mounts from the underlying
+node.
```

Comment:
This feels reasonable until we have a page about managing risk associated with `hostPath` volumes.